src/keys.rs

```diff
@@ -208,6 +217,14 @@ impl SpendValidatingKey {
             })
             .map(SpendValidatingKey)
     }
+
+    /// Attempts to convert these bytes into a spend validating key
+    /// from its serialized form, I2LEOSP_256(ak). Returns None if
+    /// it can't be created.
+    #[cfg(feature = "frost")]
+    pub fn with_bytes(bytes: &[u8]) -> Option<Self> {
+        SpendValidatingKey::from_bytes(bytes)
+    }
```

Comment:
This isn't needed if `#[cfg_attr(feature = "frost", visibility::make(pub))]` is used. In any case the method that is conditionally exposed should be called `from_bytes`.

Reply:
Cool I'll get `visibility` crate in then. Thank you!